gradio/events.py

```diff
@@ -721,3 +721,60 @@ def upload(
             every=every,
         )
         set_cancel_events(self, "upload", cancels)
+
```

Comment:
I still think we should rethink this, feels like a lot of code when it is borderline identical to all of the other events,

Reply:
Agreed 100%